dex/src/main/scala/com/wavesplatform/dex/Application.scala

```diff
@@ -563,6 +564,9 @@ object Application {
     val configFile = args.headOption
     val (config, settings) = loadApplicationConfig(configFile.map(new File(_)))
 
+    val excludedConfigKeys = config.getStringList("waves.dex.secure-keys").asScala.toSet
```

Comment:
To make this code consistent with other, let's:
1. Add the config parameter `secure-keys` to `MatcherSettings`
2. Add a check into `MatcherSettingsSpecification`. Also you have to provide a non-default value for `MatcherSettingsSpecification` in `BaseSettingsSpecification.configWithSettings`

Reply:
fixed

Reply:
So, you can use `settings.secureKeys` instead of `getStringList`.